main.py

```diff
@@ -1,36 +1,57 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+import os
 
 import sublime
 import sublime_plugin
 
 
+class DistractionFreeWindowListener(sublime_plugin.EventListener):
+    def on_activated(self, view):
+        view.window().run_command("distraction_free_window", {"toggle": False})
+
+
 class DistractionFreeWindowCommand(sublime_plugin.WindowCommand):
 
     def _status_msg(self, msg):
         self.window.status_message('Distraction Free Window: {}'.format(msg))
-
+        
     @staticmethod
     def _reset_setting(view_prefs, syntax_prefs, global_prefs, setting, default):
         view_prefs.set(setting, syntax_prefs.get(setting, global_prefs.get(setting, default)))
 
-    def run(self):
+    def run(self, toggle=True):
+        packages_path = sublime.packages_path()
+        session_file = os.path.join(os.path.dirname(packages_path), 'Local', 'Session.sublime_session')
```

Comment:
The reason was that I thought it might be better in order to avoid read/write conflicts. I don't know which one is better, but this one works for me.

Looking forward for your test results :)